README.md

```diff
@@ -1,16 +1,341 @@
-# Title
+# Object Orientation in JavaScript
+
+“Every application is a collection of code; the code’s arrangement is the
+design." - Sandy Metz
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Review the meaning of Object Orientation
+- Review the benefits of Object Orientation
+- Introduce how JavaScript can be written using Object Orientation
 
 ## Introduction
 
-## SWBAT 1
+So far in JavaScript, we've discussed the "Three Pillars of Web Programming",
+**Recognizing JS events**, **Manipulating the DOM**, and **Communicate with the
+server**, and we saw how these pillars are incorporated into web applications.
+
+The process we've learned has been straightforward:
+
+1.  An HTML page renders on the screen
+2.  JavaScript is executed when the page loads
+3.  In the JavaScript, event listeners are created pointing to 'free-standing'
+    functions, listed out in the JavaScript file.
+
```

Comment:
I think a transition sentence from list to list here would be good. "In that process, we have three primary takeaways:" or something similar.